ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+       "@type": "Alert",
+       "iCalComponent": {
+         "name": "valarm",
+         "properties": [
+           [
+             "action",
+             {},
+             "text",
+             "AUDIO"
+           ]
+         ]
+       },
+       "...": ""
+     }
+   }
+
+         Figure 18: Converting the ACTION property for value AUDIO
+
+2.3.3.  ATTACH
+
+   The ATTACH property [RFC5545] (Section 3.8.1.1) in a VEVENT, VTODO,
+   VRESOURCE, PARTICIPANT, or VLOCATION component converts to a Link
+   object [RFC8984] (Section 1.4.11).  The converted object is set in
+   the links property of the Event, Task, Participant, or Location
+   object.
+
+   The property value converts to the href property of the Link object.
+   A value of type URI converts as-is.  A value of type BINARY converts
+   to a URI in the "data" URL scheme ([RFC2397].  If the FMTTYPE
+   parameter is set on the ATTACH property, then the parameter value
+   SHOULD be set in the mediatype part of the data URL.
+
+   The ATTACH property parameters convert as follows:
+
+       +=========+==========================+===============+======+
+       | Name    | Reference                | Link Property | Note |
+       +=========+==========================+===============+======+
+       | FMTTYPE | [RFC5545], Section 3.2.8 | contentType   |      |
+       +---------+--------------------------+---------------+------+
+       | SIZE    | [RFC8607], Section 4.1   | size          |      |
+       +---------+--------------------------+---------------+------+
+
+                Table 16: Parameters of the ATTACH property
+
+   The following examples illustrate how to convert the ATTACH property:
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 39]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   ATTACH:https://example.com/foo.pdf
+   "links": {
+     "a": {
+       "@type": "Link",
+       "href": "https://example.com/foo.pdf"
+     }
+   }
+
+          Figure 19: Converting the ATTACH property with URI value
+
+   ATTACH;ENCODING=BASE64;VALUE=BINARY;FMTTYPE=image/png:iVBORw
+    0KGgoAAAANSUhEUgAAAAEAAAABAQAAAAA3bvkkAAAAAmJLR0QAAd2KE6QAA
+    AAKSURBVAjXY2gAAACCAIHdQ2r0AAAAAElFTkSuQmCC
+   "links": {
+     "a": {
+       "@type": "Link",
+       "href": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABAQAAAAA3bvkkAAAAAmJLR0QAAd2KE6QAAAAKSURBVAjXY2gAAACCAIHdQ2r0AAAAAElFTkSuQmCC",
+       "contentType": "image/png"
+     }
+   }
+
+        Figure 20: Converting the ATTACH property with BINARY value
+
+2.3.4.  ATTENDEE
+
+   The ATTENDEE property [RFC5545] (Section 3.8.4.1) in a VEVENT or
+   VTODO component converts to a Participant object [RFC8984]
+   (Section 4.4.6).  The converted object is set in the participants
+   property of the Event or Task object.
+
+   The property value converts to both the calendarAddress and sendTo
+   properties of the Participant object.  If the CAL-ADDRESS value is in
```

Comment:
There's only two references to CAL-ADDRESS, and they both say this same thing. It's unclear to me what it is or how it relates to ATTENDEE or ORGANIZER